lib/render/ControlExecutive.cpp

```diff
@@ -160,12 +160,12 @@ int ControlExec::Paint(string winName, bool fast){
 
 	// Disable state saving when generating the transfer function
 	//
-	bool enabled = _paramsMgr->GetSaveStateUndoEnabled();
-	_paramsMgr->SetSaveStateUndoEnabled(false);
+	bool enabled = _paramsMgr->GetSaveStateEnabled();
+	_paramsMgr->SetSaveStateEnabled(false);
 
 	int rc =  v->paintEvent(fast);
 
-	_paramsMgr->SetSaveStateUndoEnabled(enabled);
+	_paramsMgr->SetSaveStateEnabled(enabled);
```

Comment:
This will reintroduce #2418. I would assume the proper fix is to prevent changes performed when undo is disabled from clearing the redo list.

Reply:
I'm not sure what the "proper" fix is, but if you want to modify SetSaveStateUndoEnabled per your description we can try it.

Reply:
I checked and this does not reintroduce that bug. After some head scratching I think I figured out why this is. Also I checked and preventing the redo list from clearing when undo is disabled fixes the issue as well, however I am not familiar enough with this code to say which is the proper fix.

Reply:
Yeah, undo/redo needs some work. I've opened a ticket to do some refactoring. Hopefully the latest bandaid will get us through the release.